### ISSUE TITLE
Make footer a widgetized area

### ISSUE THREAD

**Issue by rcass** [redacademy/play-winter-2018#5]
Currently you have widgitzed area, that is not in use at all (or that I could find on the site).

![screen shot 2018-03-16 at 1 26 49 pm](https://user-images.githubusercontent.com/5028555/37543114-d32ae1ea-291d-11e8-9f95-4583e9fa2594.png)

I would only keep this if it being utilized in the blog/post functionality. If not, I would repurpose it to be a footer widgetized area.

Basically the footer should be a widgitized area since the client prob will want access to those social media links and contact info. So either create a widgetized area or just use the unused one.

#### To sum up:

- [x] Determine if 'sidebar' widgetized area is needed

- [x] Make footer widgetized area
      

- [x] Make sure social media icons in footer are clickable
